### PR TITLE
[JB][OPS-10144] add scenarios for cancelled payment journey, various …

### PR DESCRIPTION
**Pull request by barnesjake**
…little changes

### CODE REVIEW

File: app/config/ViewConfig.scala [hunk 1]
```diff
@@ -26,8 +26,6 @@ class ViewConfig @Inject() (
     val config:     Configuration,
     servicesConfig: ServicesConfig
 ) {
-  //private val contactBaseUrl = servicesConfig.baseUrl("face-to-face-frontend")
-  private val serviceIdentifier = "MyService"
```
Comment (barnesjake) [inline app/config/ViewConfig.scala:30]: Not used

File: app/actors/SpcParentActor.scala [hunk 2]
```diff
@@ -62,89 +62,19 @@ class SpcParentActor extends Actor {
...
-              ),
-              displayMessagesAuthentication = Seq(
-                (InteractionEvents.StartedEvent, InteractionPrompts.CustomerEnterPin),
-                (InteractionEvents.EventSuccess, InteractionPrompts.ProcessingTransaction),
-                (InteractionEvents.InProgress, InteractionPrompts.ConnectingToAcquirer),
-                (InteractionEvents.EventSuccess, InteractionPrompts.ProcessingTransaction),
-              )
-            )
-            context.actorOf(StandardMessageFlowUserActor.props(spcFlow), s"${transNum.value}")
+            context.actorOf(StandardMessageFlowUserActor.props(SpcFlows.successChipAndPinFlow), s"${transNum.value}")
```
Comment (barnesjake) [inline app/actors/SpcParentActor.scala:65]: I've just moved all of the flow variables defined within the pattern match statements to an object below called `SpcFlows`, I felt it was a bit nicer to read the pattern match this way. I'm happy to change it back though if anyone disagrees.

File: app/actors/SpcParentActor.scala [hunk 3]
```diff
@@ -399,3 +212,224 @@ class SpcParentActor extends Actor {
...
+    paymentResult                 = PaymentResults.cancelled,
+    receiptNodeName               = ReceiptTypeName.ReceiptType4Name,
+    transactionResult             = TranResults.SuccessResult,
+    cardVerificationMethod        = CardVerificationMethod.not_performed,
+    transactionSource             = TransactionSources.Icc,
+    displayMessagesValidation     = Seq.empty[(InteractionEvent, InteractionPrompt)],
+    displayMessagesAuthentication = Seq.empty[(InteractionEvent, InteractionPrompt)]
+  )
+
+  val cancelledByBarclaycardFlow: SpcFlow = SpcFlow(
```
Comment (barnesjake) [inline app/actors/SpcParentActor.scala:425]: This is the flow I'm not too sure about Wojciech, for scenario 2

File: app/actors/SpcParentActor.scala (hunk 2 above)
Comment (pawelpanasewicz) [reply app/actors/SpcParentActor.scala:65]: maybe by the chance; "transNum.value.toString" ?

**Review by pawelpanasewicz — DISMISSED** [review]

**Review by gitwojciech — APPROVED** [review]

**Pull request merged**